src/test/java/com/rocketden/main/service/UserServiceTests.java

```diff
@@ -35,4 +38,40 @@ public void createUserSuccess() {
         verify(repository).save(Mockito.any(User.class));
         assertEquals("rocket", response.getNickname());
     }
+
+    @Test
+    public void createUserInvalidNickname() {
+        CreateUserRequest request = new CreateUserRequest();
+        request.setNickname("rocket rocket");
+
+        assertThrows(ApiException.class, () -> {
```

Comment:
nit: Sorry if this is overkill, but I'd also suggest verifying in these two tests that the exception responses contain the correct error messages (see `RoomServiceTests.java` for examples)